media/js/firefox/new.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-;(function($, Modernizr, _gaq, site) {
+;(function($, Modernizr, dataLayer, site) {
```

Comment:
This is one of the most important and brittle scripts in bedrock. (Yes, the irony/pain there is fully understood :disappointed:).

This page needs to work flawlessly all the way down to IE 6. Much testing will be needed here.